yamux/src/connection.rs

```diff
@@ -540,12 +543,12 @@ impl<T: AsyncRead + AsyncWrite + Unpin> Active<T> {
     }
 
     fn on_drop_stream(&mut self, id: StreamId) {
-        let stream = self.streams.remove(&id).expect("stream not found");
+        let s = self.streams.remove(&id).expect("stream not found");
 
-        log::trace!("{}: removing dropped {}", self.id, stream);
-        let stream_id = stream.id();
+        log::trace!("{}: removing dropped", self.id);
```

Comment:
```suggestion
        log::trace!("{}: removing dropped stream {}", self.id, id);
```

Would make the log message more useful.